docs/en/changes/changes.md

```diff
@@ -102,7 +102,7 @@
 * Fix picking calendar with a wrong time range and setting a unique value for dashboard grid key.
 * Add PostgreSQL to Database sub-menu.
 * Implement the network profiling widget.
-* Add Nats icon for Java plugin.
+* Add component ID(132) for NAST java client plugin
```

Comment:
Move to backend part of changelog.

Reply:
`Add component ID(131) for Java Micronaut plugin` is also in UI part . 
The previous format of  micronaut plugin was also wrong , I will fix it  together  .
Finally , add mictonaut、nats icon should be in UI part , add component id for micronaut、nats in backend part

Reply:
Yes, icon is UI part. ID is backend part.